packages/client/src/runtime/utils/common.ts

```diff
@@ -65,16 +65,18 @@ export const needNamespace = {
   Decimal: 'Decimal',
 }
 
-export function needsNamespace(field: BaseField, dmmf: DMMFClass): boolean {
-  if (typeof field.type === 'string') {
-    if (dmmf.datamodelEnumMap[field.type]) {
+export function needsNamespace(fieldType: DMMF.Field['type'], dmmf: DMMFClass): boolean {
+  if (typeof fieldType === 'string') {
+    if (dmmf.datamodelEnumMap[fieldType]) {
       return false
     }
-    if (GraphQLScalarToJSTypeTable[field.type]) {
-      return Boolean(needNamespace[field.type])
+    if (GraphQLScalarToJSTypeTable[fieldType]) {
+      return Boolean(needNamespace[fieldType])
     }
   }
 
+  console.log(fieldType)
```

Comment:
Stray `console.log`

Reply:
Indeed, can someone create a PR and remove it?

Reply:
PR to remove at https://github.com/prisma/prisma/pull/10263